src/js/controller/model.js

```diff
@@ -210,6 +212,14 @@ class Model extends SimpleModel {
         }
         this.set('bandwidthEstimate', bwEstimate);
     }
+
+    _normalizeConfig(cfg) {
+        const floating = cfg.floating;
+
+        if (!floating || !!floating.disabled) {
+            cfg.floating = null;
```

Comment:
@robwalch i know we talked about using an explicit object here (`floating: { disabled: false }` when enabled `floating: { disabled: true }` when not) however given the implementation, i would have to update 3-4 checks against the model's floating config as it expects a null value currently when disabling.

i could make those updates, however i do worry about how that leads to future changes where we'll have to add more conditional logic around the `disabled` attribute in places where we're checking against it, vs just doing it once in the model and forgetting about it. thoughts?

Reply:
i suppose if we're certain we have a floating block we could remove the first half of the `floatingConfig && !floatingConfig.disabled` check. if `_normalizeConfig` were to always set a floating block could we be certain we would always have one?

Reply:
Do we have additional customizations for floating? If not maybe it should be a boolean with either floating: true, floating: false. Totally understand if existing use cases prevent this.

Reply:
`dismissable` is a config option that allows the player to be dismissed

Reply:
What do you think about doing `delete cfg.floating` rather than setting it to `null`? That would remove the property and prevent it from showing up when cloning the model attributes with `getConfig()`.

Reply:
I did the explicit null so we have a track record that it was updated by the code rather than undefined which doesn't tell us if a block was initially passed in - though we'd be able to
 get that from setup config so i think i like this approach better, will update!

Reply:
updated if you get a chance for another look @robwalch @zetagame